packages/libressl.rb

```diff
@@ -24,9 +24,15 @@ def self.install
     FileUtils.rm_rf "#{CREW_DEST_DIR}/etc/ssl"
 
     # add symlink to libcrypto.so.1.0.0
-    FileUtils.ln_s "#{CREW_LIB_PREFIX}/libcrypto.so.46.0.1", "#{CREW_DEST_LIB_PREFIX}/libcrypto.so.1.0.0"
+    FileUtils.ln_s "#{CREW_LIB_PREFIX}/libcrypto.so.1.0.0", "#{CREW_DEST_LIB_PREFIX}/libcrypto.so.45"
+
+    # add symlink to libcrypto.so.1.0.0
+    #FileUtils.ln_s "#{CREW_LIB_PREFIX}/libcrypto.so.1.0.0", "#{CREW_DEST_LIB_PREFIX}/libcrypto.so.46"
+
+    # add symlink to libssl.so.1.0.0
+    FileUtils.ln_s "#{CREW_LIB_PREFIX}/libssl.so.1.0.0", "#{CREW_DEST_LIB_PREFIX}/libssl.so.47"
 
     # add symlink to libssl.so.1.0.0
-    FileUtils.ln_s "#{CREW_LIB_PREFIX}/libssl.so.48.0.1", "#{CREW_DEST_LIB_PREFIX}/libssl.so.1.0.0"
+    #FileUtils.ln_s "#{CREW_LIB_PREFIX}/libssl.so.1.0.0", "#{CREW_DEST_LIB_PREFIX}/libssl.so.48"
```

Comment:
It doesn't hurt to have all these symlinks but a better solution would be to recompile all packages that rely on the older version to link to the newer version.